### PR TITLE
Validate passphrase (BIP39) when importing account

### PR DESCRIPTION
**Pull request by air1one**
Suggestion for issue #20 

When importing with passhprase, show an error if the passphrase is not BIP39 valid.
But allow with a switch to enter a "custom" passphrase (non-BIP39).

Just like ark-desktop : see [pull request 498](https://github.com/ArkEcosystem/ark-desktop/pull/498)

### CODE REVIEW

**Comment by air1one** [comment]
Need some rework to fix CI build.

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -227,6 +227,7 @@
     "PUBLIC_ADDRESS_SHARE": "Public Address (SHARE)",
     "PUBLIC_KEY": "Public Key",
     "PASSPHRASE_NOT_BELONG_TO_ADDRESS": "The entered passphrase doesn't belong to the address '{{ address }}'!",
+    "PASSPHRASE_NOT_BIP39": "The entered passphrase is not BIP-39 valid",
```
Comment (alexbarnsley) [inline src/assets/i18n/en.json]: I would possibly reword to: "You have not entered a valid BIP-39 passphrase"

**Comment by alexbarnsley** [comment]
Looks good! If you could reword the phrase i mentioned, I will merge 👍

**Comment by air1one** [comment]
Done 👍

**Comment by alexbarnsley** [comment]
Nice work! :+1:

**Pull request merged**